### ISSUE TITLE
Data not iterable

### ISSUE THREAD

**Issue by MikevanBreePXL** [homewizard/python-homewizard-energy#383]
It seems the data returned from `await api.data()` looks like a dictionary, but this is not iterable or able to be cast to a tuple or dict. How are we supposed to get certain key-values, if no clear methods are documented?

**Comment by MikevanBreePXL** [comment]
added comment:
It's possible to get data one-by-one by calling the key names as properties, but the issue with iterating as dict or tuple stays

**Comment by MikevanBreePXL** [comment]
Perhaps [this code commit](https://github.com/MikevanBreePXL/python-homewizard-energy/commit/8005b953864b08e7bf505df02ed1faf180920932) could be a fix, but I'm not sure how to test everything, so hereby a link to an own commit

**Comment by DCSBL** [comment]
Hi! Can you provide us with what you are trying to achieve, so we can better understand your situation?

For example, what is the goal you have in mind and how would you like that using code?

**Comment by DCSBL** [comment]
> Perhaps [this code commit](https://github.com/MikevanBreePXL/python-homewizard-energy/commit/8005b953864b08e7bf505df02ed1faf180920932) could be a fix, but I'm not sure how to test everything, so hereby a link to an own commit

I see no reason not to add this, feel free to suggest this change via a PR and I will test this ASAP!

**Comment by MikevanBreePXL** [comment]
> Hi! Can you provide us with what you are trying to achieve, so we can better understand your situation?
> 
> For example, what is the goal you have in mind and how would you like that using code?

The PR should allow complete support to turn the Data class into a dictionary.
More precisely, code like `dict_data = dict(data_var)` and `power_var = data_var["total_power_generated_kwh"]` should now be supported, as you previously only had `power_var = data_var.total_power_generated_kwh` as accepted method

**Comment by DCSBL** [comment]
While I understand what you are trying to achieve, I do not understand why you want to do that. Can you give an example of _why_ you want to access the data as a dict / iterate over it?

**Comment by MikevanBreePXL** [comment]
If you want to know the honest brutal truth...

I'm not an expert on Python, but will use it for smaller projects. When I got to coding this time around, I figured the data I had gotten back was in the form of a dictionary, given the output of a `print()`. It took me a hot minute to realize the data was saved as separate class with properties instead.

**Comment by DCSBL** [comment]
Gotcha!

No worries, I understand the situation now, and it's great to see you just go for it. We've updated the example, and I hope this helps you understand how we'd like the data to be used. Is it possible for you to adjust your solution based on that?

I've chatted with the team, and we don't think we're going to merge #386. We believe there isn't much use for it since we expect the code to be used as `<type>.<value>`, and adding it would mean some unnecessary maintenance.

Thanks a lot for your suggestion, and always feel free to share new ideas!
